docker/bigtop-puppet/build.sh

```diff
@@ -29,7 +29,14 @@ fi
 
 PREFIX=$(echo "$1" | cut -d '-' -f 1)
 OS=$(echo "$1" | cut -d '-' -f 2)
+
+# BIGTOP-4082 get the openEuler OS version
+if [ "${OS}" == "openeuler" ];then
+VERSION=$(echo "$1" | cut -d '-' -f 3-5)
+else
 VERSION=$(echo "$1" | cut -d '-' -f 3)
+fi
+
```

Comment:
How about always using `VERSION=$(echo "$1" | cut -d '-' -f 3-)` ? Since the version is the last part of the argument, we can cover all the case of `22.03` and `22.03-lts-sp1` and `22.03-lts-sp1-some-more-addendum`.

Reply:
After communicatted with the person who in charge of the openeuler OS versoin, we got the all version list of 22.03 are:
22.03-lts-sp1/22.03-lts-sp2/22.03-lts-sp3/22.03-lts-sp4, no other version number.

Reply:
The scope is not limited to openEuler. Please remove the conditional. @MacChen01

Reply:
@iwasakims 
ok, code has been modified, thanks.